src/frontend/js/components/Navigation.js

```diff
@@ -19,13 +20,36 @@ const styles = {
     marginLeft: 15,
     color: 'white'
   },
+  loggedUserLinks: {
+    clear: 'none'
+  },
   fontColor: {
     color: 'white'
   }
 }
 
 class Navigation extends React.Component {
-  render () {
+  constructor () {
+    super()
+    this.createNavBar = this.createNavBar.bind(this)
+    this.logout = this.logout.bind(this)
+  }
+
+  logout () {
+    axios.post('/api/logout')
+    .then((response) => {
+      if (response.status === 200) {
+        this.props.handleLogout()
+      } else {
+        console.log('Something went wrong. Please retry')
```

Comment:
Maybe change this to an `alert()` so that the user can see? Far from ideal, but good enough until we have HTML based error popups